### PR TITLE
Create/Use Job and deliver Attributes + File

### PR DESCRIPTION
**Pull request by darenegade**
**Description**
* Query existing jobs
* Create new job if none exists
* Add job Attributes
* Upload File

**ToDo**
- [x] Unit-Tests
- [x] ~~Request Validation~~ -> Validation is part of OceCosmos API - Errors were propagated 
- [x] File as JSON from Eventbus
- [x] Get Token
- [x] Default Values configuration

**Reference**
Issues [#307](https://github.com/it-at-m/digiwf-project/issues/307) [#306](https://github.com/it-at-m/digiwf-project/issues/306) [#308](https://github.com/it-at-m/digiwf-project/issues/308)

**Checklist**

- [x] Acceptance criteria are met
- [x] Pipeline has been run successfully
- [x] JUnit tests have been run successfully
- [ ] Changelog is adjusted
- [ ] [Documentations](https://git.muenchen.de/groups/digitalisierung/-/wikis/Dokumentationen) are completed
- [ ] Frontend is tested
- [ ] Created sub-branches are deleted
- [ ] Boards are updated ( [Digitalisierung](https://git.muenchen.de/groups/digitalisierung/-/boards) , [Support](https://git.muenchen.de/digitalisierung/digiwf-support) , [Zenhub](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) )
- [ ] feature process is created or extended
- [ ] [git-ops](https://git.muenchen.de/digitalisierung/digiwf-ops) is customized
- [ ] Openshift environments are prepared (Secrets, etc.)

### CODE REVIEW

**Comment by axd4** [comment]
consider deleting basepath from api-docs

File: digiwf-oce-cosmos-integration/src/main/java/io/muenchendigital/digiwf/ocecosmos/integration/repository/JobRepository.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    public void addJobAttributesToJob(final List<JobAttribute> attributes, Long id) throws OceCosmosIntegrationClientErrorException, OceCosmosIntegrationServerErrorException, OceCosmosIntegrationException {
+
+        catchErrors(() -> {
+            this.jobAttributeOperationsApi.putUsingPUT2(attributes, id);
+            return true; //Method needs to return something
+        }, "add Jobattributes to a Job");
+    }
+
+    /**
+     * Add {@link JobAttribute}s to a given {@link Jobs}.
```
[the remark this answers is not in the document]
Comment (darenegade) [reply digiwf-oce-cosmos-integration/src/main/java/io/muenchendigital/digiwf/ocecosmos/integration/repository/JobRepository.java]: Done

[the remark this answers is not in the document]
Comment (darenegade) [reply digiwf-oce-cosmos-integration/src/main/java/io/muenchendigital/digiwf/ocecosmos/integration/repository/JobRepository.java]: Done

**Comment by darenegade** [comment]
@axd4 I'm done with the unit test. can u please take a final look and approve

File: digiwf-oce-cosmos-integration/src/test/java/io/muenchendigital/digiwf/ocecosmos/integration/model/mapper/JobResquestMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.everyItem;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.startsWith;
+import static org.hamcrest.text.MatchesPattern.matchesPattern;
+import static org.hamcrest.core.IsNot.not;
+
+public class JobResquestMapperTest {
```
Comment (axd4) [inline digiwf-oce-cosmos-integration/src/test/java/io/muenchendigital/digiwf/ocecosmos/integration/model/mapper/JobResquestMapperTest.java:21]: Typo in ClassName

File: digiwf-oce-cosmos-integration/apidoc/api-docs.json [hunk 3]
```diff
@@ -1,4 +1,3434 @@
...
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "securityDefinitions": {
+    "oauth2schema": {
+      "type": "oauth2",
+      "tokenUrl": "http://syshubk001.srv.muenchen.de:8088/cosmos-webauth/oauth/token",
```
[the remark this answers is not in the document]
Comment (darenegade) [reply digiwf-oce-cosmos-integration/apidoc/api-docs.json:2511]: Done

File: digiwf-oce-cosmos-integration/src/test/java/io/muenchendigital/digiwf/ocecosmos/integration/model/mapper/JobResquestMapperTest.java (hunk 2 above)
Comment (darenegade) [reply digiwf-oce-cosmos-integration/src/test/java/io/muenchendigital/digiwf/ocecosmos/integration/model/mapper/JobResquestMapperTest.java]: Done

**Review by axd4 — APPROVED** [review]
Looks good. Nice job!

**Pull request merged**